hs_core/templates/search/search.html

```diff
@@ -0,0 +1,204 @@
+                    <th>Last Modified</th>
+                </tr>
+                {% if query %}
+                    {% for result in page.object_list %}
+                        <tr>
+                            <td><a href="{{ result.object.get_absolute_url }}">{{ result.object.title }}</a></td>
+                            <td>{{ result.object|resource_type }}</td>
+                            {% if result.object.first_creator.description %}
+                            <td><a href="{{ result.object.first_creator.description }}">{{ result.object.first_creator.name  }}</a></td>
+                            {% else %}
+                                <td>{{ result.object.first_creator.name }}</td>
+                            {% endif %}
+                            <td>{{ result.object.updated|date }} {{ result.object.updated|time }}</td>
+                        </tr>
+                    {% empty %}
+                    <p>No results found.</p>
+                    {% endfor %}
+                {% endif %}
+            </table>
+        </div>
+    </div>
+</div>
+{% endblock %}
+
+
+{% block extra_js %}
+    <script type="text/javascript">
+    var reorderDivs = function(){
+        var faceted_fields = ['author', 'subjects', 'resource_type', 'public', 'creators', 'discoverable', 'language'];
+         var div_ordering = [];
+         faceted_fields.forEach(function(field){
+             var faceting_div = "faceting-"+field;
+             div_ordering.push(faceting_div);
+         });
+         var i;
+         for (i = 1; i < div_ordering.length; i++) {
+             var div0 = "#" + div_ordering[i-1];
+             var div1 = "#" + div_ordering[i];
+             $(div1).insertAfter(div0);
+         }
+    }
+
+    var clearAllFacted = function(){
+        window.localStorage.clear();
+        var textSearch = $("#id_q").val();
+        var clearURL = "/search/?q=" + textSearch;
+        window.location = clearURL;
+    }
+
+     $(document).ready(function () {
+         $("#id_q").attr('placeholder', 'Search...');
+         reorderDivs();
+         $('.collapse').on('shown.bs.collapse', function(){
+            $(this).parent().find(".glyphicon-plus").removeClass("glyphicon-plus").addClass("glyphicon-minus");
+         }).on('hidden.bs.collapse', function(){
+             $(this).parent().find(".glyphicon-minus").removeClass("glyphicon-minus").addClass("glyphicon-plus");
+         });
+
+         $("#id_q").keypress(function(event) {
+            if (event.which == 13) {
+                event.preventDefault();
+                $("form").submit();
+            }
+         });
+         $(function () {
+             $(".faceted-selections").click(function () {
+                console.log($(this).val());
+                var tokens = $(this).val().split(",");
+                var key = tokens[0];
+                var value = tokens[1];
+                var requestURL = "{{ request.get_full_path }}"
```

Comment:
Rather than mixing Javascript and Django template syntax, just use the existing value of `window.location.pathname`